pkg/constant/constant_shared_test.go

```diff
@@ -127,6 +127,22 @@ func TestRuncModuleVersions(t *testing.T) {
 	)
 }
 
+func TestHelmModuleVersions(t *testing.T) {
```

Comment:
There is no any reason to have this test. The helm binary we have is not related to the k0s binary, it is just for the use in one of the e2e tests, has 0 connection with the module path.

Reply:
It's true that there's no direct connection between the binary used in the ITs and the Go module version. On the other hand, it might be confusing to have the versions diverge. Folks might look at the Makefiles and might make false assumptions about the version that's being used in k0s itself, or the helm_version variable will be picked up in the future by something that really has an impact on k0s.

OTOH, I'd be fine to remove the variable and this test and inline the version directly into the Dockerfile, as we do it e.g. for cri-dockerd.